src/bindings/python/tests/test_runtime/test_infer_request.py

```diff
@@ -347,21 +351,22 @@ def check_fill_inputs(request, inputs):
     check_fill_inputs(request, inputs)
 
 
+@pytest.mark.skipif(os.environ.get("TEST_DEVICE", "CPU") != "CPU",
+                    reason=f"Cannot run test on device {os.environ.get('TEST_DEVICE')}, Plugin specific test")
 def test_infer_mixed_keys(device):
```

Comment:
may be we need to add info why tests is device specific? Code itself looks generic